tests/system/action/motion/test_create.py

```diff
@@ -290,6 +290,33 @@ def test_create_missing_origin_id(self) -> None:
             "data must not contain {'origin_id'} properties" in response.json["message"]
         )
 
+    def test_create_lead_motion(self) -> None:
+        self.set_models(self.permission_test_model)
+        self.set_models(
+            {
+                "meeting/1": {"name": "meeting_1 _test"},
+                "motion/3": {"meeting_id": 1, "category_id": 12, "block_id": 13},
+                "motion_category/12": {"meeting_id": 1},
+                "motion_block/13": {"meeting_id": 1},
+            }
+        )
+        response = self.request(
+            "motion.create",
+            {
+                "title": "test_Xcdfgee",
+                "meeting_id": 1,
+                "workflow_id": 12,
+                "text": "test",
+                "lead_motion_id": 3,
+                "block_id": 13,
```

Comment:
Please make two tests from this single one:
- The request user have not motion.can_manage. Do not give `block_id` and `category_id`. They must be taken from the lead motion.
- The request user has motion.can_manage. Do not give `block_id` and `category_id`. They must not be taken from the lead motion and be empty.

Reply:
The first point is the `test_create_permission_amendment`
Add another test for the second point: `test_create_permission_amendment_2`